chess/4-database/database.md

```diff
@@ -114,19 +114,70 @@ As part of your unit test deliverable you need to meet the following requirement
 
 For this phase the TAs will grade the quality of your project's source code. The rubric used to evaluate code quality can be found here: [Rubric](../code-quality-rubric.md)
 
+### Common Submission Issues
+
+Since you're adding code that acts with an external program in Phase 4, the likelihood of having auto-grader test
+output different from your local output is higher than previous phases.
```

Comment:
Was the line break here intentional?

Reply:
My editor didn't have softwrapping turned on. Now I have and will undo these explicit line breaks.